### DISCUSSION TITLE
I always get invalid access message

### DISCUSSION THREAD

**Discussion by glauberssoutodossantos** [chatwoot/chatwoot#4859] [Chatwoot APIs] [answered]
Hi, I need a help with Chatwoot API

I'm trying to integrate my system with chatwoot for a POC.

But using the documentation that is on the site, I can't make any calls, always returning invalid access.

![unknown](https://user-images.githubusercontent.com/99494469/173585314-6c75fb10-9105-43e8-92f6-20872283d73b.png)

**Comment by sojan-official** [comment, marked as answer]
@glauberssoutodossantos  platform API access is only allowed on self-hosted installations. Also, you would need to use `platform_app` access tokens instead of user tokens. 

if you are trying to access agents, use application APIs instead
https://www.chatwoot.com/developers/api/#tag/Agents

**Reply by glauberssoutodossantos** [reply]
Thanls @sojan-official. How can i get platform_app access tokens?

**Reply by glauberssoutodossantos** [reply]
i'm asking because i have a self-hosted installations and a chatwoot cloud demo account.

In the demo account I was able to search for contact information, as shown in the print below.

![image](https://user-images.githubusercontent.com/99494469/173609082-62775e03-b667-47af-8ca9-5383b718b9a2.png)


Already in my self-hosted installations app it asks for login information. I 

![image](https://user-images.githubusercontent.com/99494469/173609251-f03b9d43-2a17-436c-919e-778f68fa8564.png)

Maybe I'm also using the wrong URL for API, but I honestly didn't understand how to get the correct url.

**Reply by sojan-official** [reply]
if you are using a self-hosted installation that involves Nginx. please check this case https://github.com/chatwoot/chatwoot/issues/4773#issuecomment-1152324047
